### PR TITLE
Contribute GH Build action to platform.ui

### LINKED ISSUE #150: Contribute GH build action
Would be nice to add a GH build action to platform.ui

### PR DESCRIPTION
**Pull request by vogella**
Adds GH verification in the hope that GH validation is faster than the
Jenkins one.

Fixes #150

### CODE REVIEW

**Comment by laeubi** [comment]
You should add the unitest.yml as well ...

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        mvn
+        --batch-mode
+        -Pbuild-individual-bundles
+        -Pbree-libs
+        -Papi-check
+        -Dcompare-version-with-baselines.skip=false
+        -Dproject.build.sourceEncoding=UTF-8
+        -Dorg.slf4j.simpleLogger.showDateTime=true 
+        -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss.SSS
+        -DtrimStackTrace=false
```
Comment (laeubi) [inline .github/workflows/build.yml:39]: You might want to add property t not fail on failing tests and using multiple threads.

Comment (vogella) [reply .github/workflows/build.yml:39]: I tried to use the same settings as Jenkins build file. We should change both if we want to but I would avoid doing this for the intial contribution.

Comment (laeubi) [reply .github/workflows/build.yml:39]: do not failed with  test ist a requirement for the unitest action to work.

**Comment by laeubi** [comment]
You are missing the toolcahins.xml and toolchain maven option.

**Comment by vogella** [comment]
> You are missing the toolcahins.xml and toolchain maven option.

As does the Jenkins file

**Comment by akurtakov** [comment]
> > You are missing the toolcahins.xml and toolchain maven option.
> 
> As does the Jenkins file

JIPP auto inject the toolchain.xml so no need to handle it manually.

**Comment by vogella** [comment]
> > > You are missing the toolcahins.xml and toolchain maven option.
> > 
> > 
> > As does the Jenkins file
> 
> JIPP auto inject the toolchain.xml so no need to handle it manually.

Ignore my deleted comment, I saw now that it was silly.

**Comment by laeubi** [comment]
> Would be nice to have the complete build command in Jenkins build file too

Why? it doesn't make sense, as it *is* the complete commandline as jenkins uses some default locations that maven look at...

And it would harm because if you specify it manually maven would complain that the file do not exits :-)

Hopefully we will get 
- https://github.com/actions/setup-java/pull/282

so toolchains are setup automatically at GH, so please vote/comment/complain/... there to get more attention to this PR :-)

**Comment by vogella** [comment]
Thanks @laeubi and @akurtakov

**Pull request merged**